lib/ide/src/controller/text.rs

```diff
@@ -0,0 +1,150 @@
+//! Text Controller module.
+//!
+//! Facade over filesystem API or module text API for text editor. Does discerning between Luna
+//! module file and plain text file. In case of luna module idmap and metadata are hidden for the
+//! user.
+
+use crate::prelude::*;
+
+use crate::controller;
+
+use failure::_core::fmt::{Formatter, Error};
+use flo_stream::{Publisher, Subscriber};
+use flo_stream::MessagePublisher;
+use file_manager_client as fmc;
+use json_rpc::error::RpcError;
+use shapely::shared;
+
+
+
+// ====================
+// === Notification ===
+// ====================
+
+/// A buffer size for notification publisher.
+///
+/// We don't expect much traffic on file notifications, therefore there is no need for setting big
+/// buffers.
+const NOTIFICATION_BUFFER_SIZE : usize = 36;
```

Comment:
What happens if we exceed that? Panic will be raised? If not, what would happen? This information should be incliuded in the readme.

Reply:
I add, although such information should be looked for rather in flo_stream library documentation.